### PR TITLE
Fixes #37360 - Upgrade foreman-vendor packages to 13.x

### PR DESCRIPTION
**Pull request by chris1984**
@ekohl here is the Foreman PR to go along with https://github.com/theforeman/foreman-packaging/pull/10682

### CODE REVIEW

**Comment by ShimShtein** [comment]
I think you need the whole thing, like in https://github.com/theforeman/foreman/commit/7008091a7245a89f98ce459751d0e7b5fe08db95.

**Comment by chris1984** [comment]
@ShimShtein pulled in the cherrypick from Samir instead of doing it manually.

File: package.json [hunk 1]
```diff
@@ -20,7 +20,12 @@
     "analyze": "./script/webpack-analyze"
   },
   "dependencies": {
+<<<<<<< HEAD
     "@theforeman/vendor": "^12.0.1",
+=======
+    "@module-federation/utilities": "^1.7.0",
+    "@theforeman/vendor": "^13.0.1",
+>>>>>>> 7008091a7 (Fixes #37312 - Upgrade foreman-vendor packages to 13.x)
```
Comment (evgeni) [inline package.json]: this doesn't look correct, and we don't need module federation utilities in 3.9

File: package.json [hunk 2]
```diff
@@ -20,7 +20,12 @@
     "analyze": "./script/webpack-analyze"
   },
   "dependencies": {
+<<<<<<< HEAD
```
Comment (ShimShtein) [inline package.json]: Missed merge?

**Comment by chris1984** [comment]
@ShimShtein sorry about that, fixed it up.

File: package.json (hunk 1 above)
Comment (chris1984) [reply package.json]: That was def my fault, I should have caught the `=====` and the `>>>` fixed up

**Review by ekohl — APPROVED** [review]

**Pull request merged**